sampler.py

```diff
@@ -296,7 +297,8 @@ def iterate(self, model, useOracle):
             self.initOptima(self.scores, self.energy, self.std_dev)
 
         self.F, self.DE = self.getDelta(self.scores)
-        self.acceptanceRatio = np.minimum(1, np.exp(-self.DE / self.temperature))
+       
+        self.acceptanceRatio = np.minimum(1, np.exp(-self.DE / torch.as_tensor(self.temperature)))
```

Comment:
does this need to be tensor? it's weird that it is mixed with np. why did you change that?

Reply:
TypeError: unsupported operand type(s) for /: 'Tensor' and 'list'